### DISCUSSION TITLE
Custom Pagination

### DISCUSSION THREAD

**Discussion by Yonas-Gashaw** [NG-ZORRO/ng-zorro-antd#6183] [Q&A] [answered]
Can anyone please share me a thought how to customize the style for the default pagination of the ant design?
I want it, for instance, to be like - 10 per page instead of 10 / page.
Note that I'm using it in my current angular project.

Thanks!

**Comment by stygian-desolator** [comment, marked as answer]
Hi~
`nz-pagination` use `nz-select` as its `nz-pagination-options` 
see https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/components/pagination/pagination-options.component.ts#L89

If you really want to modify the content shown here, 
you can try this： https://stackblitz.com/edit/angular-ng6zes?file=src/app/app.module.ts
![image](https://user-images.githubusercontent.com/73412177/102002651-57c6ab00-3d39-11eb-9372-49cde4e7b78a.png)

it may help you

**Reply by vthinkxie** [reply]
Hi, @stygian-desolator it would be great if you can submit a PR to add this in the doc https://ng.ant.design/docs/i18n/en

**Reply by stygian-desolator** [reply]
> Hi, @stygian-desolator it would be great if you can submit a PR to add this in the doc https://ng.ant.design/docs/i18n/en

I am glad to submit a PR, but I think it is more appropriate to add this part of the content in the [FAQ](https://ng.ant.design/docs/faq/en) 😄

**Reply by vthinkxie** [reply]
I mean some docs like how to overwrite i18n configuration, it would also be ok to add it in the FAQ
